x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -203,19 +210,28 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
     setCellProps: any;
   }) => {
     const cellValue = columnsData[rowIndex][columnId];
+    const actualCount = columnsData[rowIndex] && columnsData[rowIndex].actual_class_doc_count;
+    // @ts-ignore
+    let accuracy;
+
+    if (columnId !== ACTUAL_CLASS_ID && actualCount) {
+      accuracy = cellValue / actualCount;
+      // round to 2 decimal places without converting to string;
+      accuracy = Math.round(accuracy * 100) / 100;
+      accuracy = `${Math.round(accuracy * 100)}%`;
+    }
     // eslint-disable-next-line react-hooks/rules-of-hooks
     useEffect(() => {
       if (columnId !== ACTUAL_CLASS_ID) {
         setCellProps({
           style: {
-            backgroundColor: `rgba(0, 179, 164, ${cellValue})`,
+            // @ts-ignore
+            backgroundColor: `rgba(0, 179, 164, ${accuracy})`,
           },
         });
       }
     }, [rowIndex, columnId, setCellProps]);
```

Comment:
I suspect we need to add `accuracy` here to the `useEffect` comparator.

Reply:
Accuracy is derived from the values we get from `rowIndex` and `columnId` - which are both in the comparator - so as far as I can tell we shouldn't need to add accuracy.